src/psyclone/domain/common/algorithm/psyir.py

```diff
@@ -279,8 +338,7 @@ def node_str(self, colour=True):
         :rtype: str
 
         '''
-        return "{0}[name='{1}']".format(
-            self.coloured_name(colour), self.symbol.name)
+        return f"{self.coloured_name(colour)}[name='{self.symbol.name}']"
```

Comment:
This is almost the same (= to :) from the super implementation, can we delete this?

Reply:
Yes, well spotted. I guess there is a more general issue of whether we should use ':' or '=' for consistency but that is to do with the parent classes so is separate to this PR.